etl/steps/data/garden/faostat/2022-05-17/shared.py

```diff
@@ -1725,7 +1754,18 @@ def optimize_table_dtypes(table: catalog.Table) -> catalog.Table:
         if c in table.columns
     }
 
-    return catalog.frames.repack_frame(table, dtypes=dtypes)
+    # Store variables metadata before optimizing table dtypes (otherwise they will be lost).
```

Comment:
(This is probably something that can happen implicitly in `repack_frame` function, but keep it as it is)